editor/test/integration/test_util.clj

```diff
@@ -529,8 +529,12 @@
   (get-in (g/node-value node-id :_properties) [:properties label :error]))
 
 (defn prop! [node-id label val]
-  (let [[node-id label] (resolve-prop node-id label)]
-    (g/set-property! node-id label val)))
+  (let [prop (get-in (g/node-value node-id :_properties) [:properties label])]
+    (if-let [set-fn (-> prop :edit-type :set-fn)]
```

Comment:
Should we maybe do the same for the `:get-fn` and `:clear-fn` in `prop` and `prop-clear!`? I see you partially fixed `with-prop`, but that will not use the `:get-fn` currently.

Reply:
I implemented `:clear-fn` support, but didn't find anything about `:get-fn`s in properties (except in the properties view for creating derived controls)